src/components/ui/Accordion/AccordionItem.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { forwardRef } from 'react'
+import type { HTMLAttributes } from 'react'
+import {
+  Icon as UIIcon,
+  AccordionItem as UIAccordionItem,
+  AccordionPanel as UIAccordionPanel,
+  AccordionButton as UIAccordionButton,
+} from '@faststore/ui'
+
+interface Props extends HTMLAttributes<HTMLDivElement> {
```

Comment:
I wonder if it's a good idea to extend from `AccordionItemProps` (imported from `@faststore/ui`)?

Same comment for `Accordion`'s and `AccordionProps` https://github.com/vtex-sites/base.store/blob/6b872ab34aad9927eff221930f00f1791fe18cd7/src/components/ui/Accordion/Accordion.tsx#L5

Reply:
Good idea, thanks!!

Reply:
It seems like a good idea to me. Very straightforward.